background/rust.tex

```diff
@@ -421,8 +422,8 @@ \subsection{Zero-cost Abstractions}
 
 There is still one vital difference between the two examples.
 We can safely have multiple references to our vector in {\Java} without worrying about what would happen if its memory would grow and reallocate.
-However, we can end up with dangling pointers if we happen to have multiple references to our vector in {\Cpp}, and it grows and reallocates while we are holding such a reference.
-\todo{Does this statement require further explanation? (i.e. explain why a reference to a stack-allocated vector in {\Cpp} can lead to a dangling pointer?}
+However, we can end up with dangling pointers if we happen to have multiple references into our vector in {\Cpp}.
+This is the case because, when the vector grows and gets reallocated, references into the vector will not be updated to the newly allocated storage but will instead be dangling pointers into the now old and deallocated storage.
```

Comment:
Nice!

Forslag; splitt til to setninger?
"This is the case because, when the vector grows and gets reallocated, references into the vector will not be updated to the newly allocated storage.
These references will instead be dangling pointers into the now old and deallocated storage."